firmware/console/binary/tooth_logger.cpp

```diff
@@ -78,23 +81,38 @@ int copyCompositeEvents(CompositeEvent *events) {
 static void SetNextCompositeEntry(efitick_t timestamp, bool trigger1, bool trigger2,
 		bool isTDC DECLARE_ENGINE_PARAMETER_SUFFIX) {
 	uint32_t nowUs = NT2US(timestamp);
+	
 	// TS uses big endian, grumble
 	buffer[NextIdx].timestamp = SWAP_UINT32(nowUs);
 	buffer[NextIdx].priLevel = trigger1;
 	buffer[NextIdx].secLevel = trigger2;
 	buffer[NextIdx].isTDC = isTDC;
 	buffer[NextIdx].sync = engine->triggerCentral.triggerState.shaft_is_synchronized;
-	buffer[NextIdx].coil = coil;
-	buffer[NextIdx].injector = injector;
+	//ptr_buffer[NextIdx].coil = coil;
```

Comment:
why do we add dead code? please add code comments explaining need for dead code

Reply:
i forgot to uncomment those, already did it.